tests/cephfs/cephfs_top/validate_fs_top_stats.py

```diff
@@ -0,0 +1,368 @@
+            )
+            return 1
+        client1 = clients[0]
+        client2 = clients[1]
+        for client in [client1, client2]:
+            setup_fs_top(client)
+
+        fs_name = "cephfs-top" if not erasure else "cephfs-top-ec"
+        fs_details = fs_util.get_fs_info(client1, fs_name)
+        if not fs_details:
+            fs_util.create_fs(client1, fs_name)
+
+        fs_util.auth_list([client1])
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        log.info("Scenario 1: cephfs-top should be empty when no clients have mounted")
+        out, rc = client2.exec_command(sudo=True, cmd=f"cephfs-top --dumpfs {fs_name}")
+        top_details = json.loads(out)
+        is_empty = (
+            "Empty"
+            if isinstance(top_details.get(fs_name), dict) and not top_details[fs_name]
+            else "Not Empty"
+        )
+        if is_empty != "Empty":
+            raise CommandFailed(
+                f"top is populating data with out mounting the clients, Popupated data : {top_details}"
+            )
+
+        client1.exec_command(sudo=True, cmd="yum install -y --nogpgcheck ceph-fuse")
+
+        log.info(
+            "Scenario 2: Cephfs-top should have details of the mount after running the writing the data using "
+            "both clients"
+        )
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ip = fs_util.get_mon_node_ips()
+        mon_node_ip = ",".join(mon_node_ip)
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_1,
+            mon_node_ip,
+            extra_params=f",fs={fs_name}",
+        )
+        active_mds = fs_util.get_active_mdss(client1, fs_name)
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph tell mds.{active_mds[0]} client ls --format json"
+        )
+        client_details = json.loads(out)
+        kernel_client_ls = [item["id"] for item in client_details]
+        kernel_client_id = kernel_client_ls[0]
+        fs_util.fuse_mount(
+            [client1], fuse_mounting_dir_1, extra_params=f" --client_fs {fs_name}"
+        )
+
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph tell mds.{active_mds[0]} client ls --format json"
+        )
+        client_details = json.loads(out)
+        fuse_client_ls = [item["id"] for item in client_details]
+        fuse_client_id = [
+            item for item in fuse_client_ls if item not in kernel_client_ls
+        ][0]
+        log.info(f"fuse_client_id : {fuse_client_id}")
+
+        with parallel() as p:
+            p.spawn(fs_util.run_ios, client1, fuse_mounting_dir_1, ["dd", "smallfile"])
+            p.spawn(
+                fs_util.run_ios, client1, kernel_mounting_dir_1, ["dd", "smallfile"]
```

Comment:
Is this scenario supposed to run on 2 different clients.  ? As the Test Scenario mentions 2 clients and both the dirs are mounted and run from same client.